UnicodeJsps/src/main/java/org/unicode/jsp/UnicodeUtilities.java

```diff
@@ -1440,6 +1442,42 @@ public static void showProperties(
 
         String kRSUnicode = getFactory().getProperty("kRSUnicode").getValue(cp);
         boolean isUnihan = kRSUnicode != null;
+        List<UcdProperty> indexedProperties =
```

Comment:
optional -- might simplify something:

How about, rather than just building separate lists of properties, you add an `enum PropCategory { UCD, NON_UCD, ... CJK, ...}`, and create a `Map<PropCategory, List<UcdProperty>>`?

You could then also use maps from PropCategory to table headings and such.

----
Does it matter if these lists are List's? Or do you just need Collection's?

Reply:
> Does it matter if these lists are List's?

Not really, I convert them to lists of String below anyway (because one of them is not a list of UcdProperty, namely the list of stuff that gets added in the tools).

Reply:
I will merge this now to unblock John and see if I can come up with something cleaner in a subsequent PR.